app/client/views/resumes/resumesCtrl.js

```diff
@@ -0,0 +1,425 @@
+const moment = require('moment');
+const swal = require('sweetalert');
+const pdfjsLib = require('pdfjs-dist');
+
+// The workerSrc property shall be specified.
+pdfjsLib.GlobalWorkerOptions.workerSrc = `//cdnjs.cloudflare.com/ajax/libs/pdf.js/${pdfjsLib.version}/pdf.worker.js`;
+
+angular.module('reg')
+  .controller('ResumesCtrl', [
+    '$scope',
+    '$state',
+    '$stateParams',
+    'UserService',
+    function ($scope, $state, $stateParams, UserService) {
+
+      $scope.pages = [];
+      $scope.users = [];
+
+      // Semantic-UI moves modal content into a dimmer at the top level.
+      // While this is usually nice, it means that with our routing will generate
+      // multiple modals if you change state. Kill the top level dimmer node on initial load
+      // to prevent this.
+      $('.ui.dimmer').remove();
+      // Populate the size of the modal for when it appears, with an arbitrary user.
+      $scope.selectedUser = {};
+      $scope.selectedUser.sections = generateSections({
+        status: '',
+        confirmation: {
+          dietaryRestrictions: []
+        },
+        profile: ''
+      });
+
+      function updatePage(data) {
+        $scope.users = data.users;
+        $scope.currentPage = data.page;
+        $scope.pageSize = data.size;
+
+        var p = [];
+        for (var i = 0; i < data.totalPages; i++) {
+          p.push(i);
+        }
+        $scope.pages = p;
+      }
+
+      UserService
+        .getPage($stateParams.page, $stateParams.size, $stateParams.query)
+        .then(response => {
+          updatePage(response.data);
+        });
+
+      $scope.$watch('queryText', function (queryText) {
+        UserService
+          .getPage($stateParams.page, $stateParams.size, queryText)
+          .then(response => {
+            updatePage(response.data);
+          });
+      });
+
+      $scope.goToPage = function (page) {
+        $state.go('app.admin.users', {
```

Comment:
Should this be changed to app.resumes.users or something of that nature?

Reply:
what does this actually do? I tried changing it to some random gibberish but it still works normally lol

Reply:
changed